test/TestExpectations.json

```diff
@@ -1331,6 +1331,12 @@
     "parameters": ["chrome", "webDriverBiDi"],
     "expectations": ["FAIL", "TIMEOUT"]
   },
+  {
+    "testIdPattern": "[accessibility.spec] Accessibility should work",
```

Comment:
Due to use of latest Canary we are seeing different Accessibility tree: 
This is now not reported, maybe a bug,
```json
{role: 'StaticText', name: 'Hello World'}
```
All the other test have more info so we should ignore them that info for now via `toMatchObject`